app/Main.hs

```diff
@@ -96,16 +96,18 @@ printDirectory (DirectoryPrefix dir) = do
     setSGR   [Reset]
 
 printTermResults :: Int -> (String, TermResults) -> IO ()
-printTermResults w (_, results) = do
+printTermResults w (_, results) =
     printMatches w results $ matches results
 
+likelihoodColor :: RemovalLikelihood -> Color
+likelihoodColor High = Red
```

Comment:
What would the guard look like here? It seems like it would be more verbose, because you'd need `==` for each member.